src/sass/components/_org-cards.scss

```diff
@@ -13,20 +13,38 @@
     padding-left: 0;
     padding-right: 0;
   }
-
-  .slds-media {
-    margin-bottom: $spacing-medium;
-  }
 }
 
 // override min-width and force header to wrap on nested card headers
 .wrap-inner-truncate {
   .slds-card__header {
+    align-items: baseline;
     flex-wrap: wrap;
+
+    // Specifically target OrgActions in card header
+    .slds-no-flex {
+      align-items: baseline;
+      display: flex;
+      flex: 0 1 auto;
+      flex-wrap: wrap; // allow button group to wrap when space is limited
+      margin-right: $spacing-medium;
```

Comment:
I assume this is leaving space for the `metecho-card-more` dropdown? Maybe worth commenting to clarify that the sizes are related?

Reply:
@mirisuzanne I made a few changes since your comment as it made me think about how to simplify and keep the margin consistent. Instead, I edited the padding end variable to give the button enough space on the right instead of larger right margins on the title container and button containers (they both now have a small margin which looks better anyway) :)